File: 1768. Merge Strings Alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        int i = 0;
        int j = 0;
        int n1 = word1.length();
        int n2 = word2.length();
        String res = "";
        while (i < n1 || j < n2){
            if(i < n1)
                res += word1.charAt(i);
            if(j < n2)
                res += word2.charAt(j);
            i += 1;
            j += 1;
        }
        return res;
    }
}

File: 1768. Merge Strings Alternately.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        n = len(word1)
        m = len(word2)
        i = j = 0
        ans = ""
        while i < n and j < m:
            ans = ans + word1[i]
            ans = ans + word2[j]
            i += 1
            j += 1
        while i < n:
            ans = ans + word1[i]
            i += 1
        while j < m:
            ans = ans + word2[j]
            j += 1
        return ans